### PR TITLE
see list of users on team 164484578

### PR DESCRIPTION
**Pull request by justgarcia5**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -14,8 +14,10 @@
         <div class="nav-link">
           <% if current_user.is_admin? %>
             <%= link_to "User List", users_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'users'}" %>
-            <%= link_to "Initiatives", root_path, class: "nav-btn #{"nav-btn-color" if controller_name == 'initiatives'} " %>
+            <% elsif current_user.team_members.present? %>
```
Comment (jasononaquest) [inline app/views/layouts/application.html.erb]: This should probably be outdented to the same level as the `if`.

File: app/views/users/index.html.erb [hunk 2]
```diff
@@ -1,6 +1,8 @@
 <div class="wrapper">
   <div class="add-user-btn">
+    <% if current_user.is_admin %>
     <%= link_to 'Add User', new_user_path, class: 'btn-primary w-button' %>
```
Comment (jasononaquest) [inline app/views/users/index.html.erb]: It can be helpful to indent blocks within an `if-else`.

File: config/routes.rb [hunk 3]
```diff
@@ -2,6 +2,7 @@
 
 Rails.application.routes.draw do
   devise_for :users, :controllers => { :sessions => "sessions" }
+  get '/users/team_members', to: 'users#team_members', as: 'team'
```
Comment (jasononaquest) [inline config/routes.rb]: Is there a path that makes more sense than `/users/team_members` for this route? Using `as` suggests you think `/team` makes more sense. How could you rewrite this without the `as` to achieve that?

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (justgarcia5) [reply app/views/layouts/application.html.erb]: We resolved this issue

File: app/views/users/index.html.erb (hunk 2 above)
Comment (justgarcia5) [reply app/views/users/index.html.erb]: This issue was resolved

File: config/routes.rb (hunk 3 above)
Comment (justgarcia5) [reply config/routes.rb]: The path was changed to /team

**Review by jasononaquest — APPROVED** [review]
LGTM

**Pull request merged**